task/tekton-catalog-publish/0.1/tekton-catalog-publish.yaml

```diff
@@ -0,0 +1,70 @@
+---
+apiVersion: tekton.dev/v1beta1
+kind: Task
+metadata:
+  name: tekton-catalog-publish
+  labels:
+    app.kubernetes.io/version: "0.1"
+  annotations:
+    tekton.dev/pipelines.minVersion: "0.17.0"
+    tekton.dev/tags: catalog, bundles
+    tekton.dev/displayName: "Publish a Tekton Catalog"
+spec:
+  description: >-
+    This task publishes a Tekton catalog as bundles to a container registry
```

Comment:
The PR template recommends an explanatory paragraph. The readme has some stuff that could go here.

Reply:
done